README.markdown

```diff
@@ -2,6 +2,12 @@
 
 A simple way to manipulate images "just in time" via the URL. Supports caching, image quality settings and loading of offsite images.
 
+-----
+
+**NOTE: Version 2.x is not working properly, please use the 1.x branch until further notice!**
+
```

Comment:
Can you add this link https://github.com/symphonycms/jit_image_manipulation/tree/1.x please ?